Porfolio :Finish Mobile Version

NoelLincoln opened the pull request:
Add works section and about myself section
Use grid layout to place card elements in a colum and flexbox to place elements within the card element
Use JQuery toggle function on dropdown list to show and hide languages

NoelLincoln committed:
Add work card section:
    Use flexbox to align the main div elements in a column
    Use flexbox to align the inner div elements in rows and columns
    Style text with the right fonts and font sizes as from figma

NoelLincoln committed:
Add more markup and styling:
    Align divs properly using flex property
    Add the skills section and social media links
    Give text font as from figma design

NoelLincoln committed:
Add feature: dropdown list and fixing fonts
    style dropdown list for languages using flexbox
    set toggle visibility for dropdown items from none to block onclick using JQuery

NoelLincoln committed:
Fix dropdown responsivenes:
    use pixels in width for dropdown content

NoelLincoln committed:
Add project cards to works section
    use grid display property to align projects in a column

NoelLincoln committed:
fix linter errors on css

Efiamotu-1 commented on index.html:
```diff
@@ -71,8 +74,289 @@ <h1>I’m Noel Bryant from Kenya. Glad to see you!</h1>
+
+            <ul class="dropdown-content" role="menu">
+              <li><img src="assets/images/javascript.png" alt="javascript
+                  image"/><a href="#">Javascript</a></li>
+              <li><img src="assets/images/html.png" alt="html
+                  image"/><a href="#">HTML</a></li>
+              <li><img src="assets/images/javascript.png" alt="javascript
+                  image"/><a href="#">CSS</a></li>
+            </ul>
+          </div>
+          <div class="dropdown">
+            <input type="checkbox" id="dropdown" />
+            <label for="dropdown" class="dropdown-btn">
+              <span class="languages">Frameworks</span>
+              <span class="arrow right"></span>
+            </label>
+
+            <ul class="dropdown-content-frameworks" role="menu">
+              <li><a href="#">React</a></li>
+              <li><a href="#">Angular</a></li>
+              <li><a href="#">Vue</a></li>
+              <li><a href="#">Svelte</a></li>
+            </ul>
+          </div>
+          <div class="dropdown">
+            <input type="checkbox" id="dropdown" />
+            <label for="dropdown" class="dropdown-btn">
+              <span class="languages">Skills</span>
+              <span class="arrow right"></span>
+            </label>
+
+            <ul class="dropdown-content-skills" role="menu">
+              <li><a href="#">React</a></li>
+              <li><a href="#">Angular</a></li>
+              <li><a href="#">Vue</a></li>
+              <li><a href="#">Svelte</a></li>
+            </ul>
+          </div>
+        </div>
+
```
- [ ] Your technology categories such as `languages, frameworks and skills` should be wrapped in `ul` and `li` tags. It is important to make use of semantic list item tags to display the list of technologies. :+1:

Efiamotu-1 commented on index.html:
- [ ] I want to kindly suggest that you add the button interactions on all buttons such as `hover, active` as provided by the figma template below. It is important to add interactivity to your web page as it makes it more functional and interactive to users who visit it :+1: :sparkles: 
- button interaction
![b-i](https://github.com/NoelLincoln/mportfolio/assets/89055502/b5383343-1f0e-475e-b261-0ce726c75b06)

Efiamotu-1 reviewed (changes requested):
Hi @NoelLincoln , 

![](https://user-images.githubusercontent.com/89055502/221963929-dc172827-5c2c-411b-bf4d-923659f626e1.gif) 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [![@Efiamotu-1](https://img.shields.io/github/followers/Efiamotu-1?label=Habeeb-Efiamotu-Musa&style=social)](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [![@EFYAMOTU](https://img.shields.io/twitter/follow/omarramoun?label=Habeeb-Efiamotu-Musa&style=social)](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[![Habeeb Efiamotu Musa](https://img.shields.io/badge/linkedin-@Habeeb-crimson)](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [![Habeeb Efiamotu Musa ](https://img.shields.io/badge/slack-@Habeeb-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NoelLincoln committed:
Fix layout and use semantic tags
    use semantic tags for titles and lists
    reduce over-spaced elements and match them closely to figma design
    add button interaction on hover and click

NoelLincoln committed:
fix linter errors in css

DuaneDave reviewed (approved):
Hi @NoelLincoln 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ All issues completely addressed
✔️ Good job adding the dropdown functionality, so awesome

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
